utils/filesystem/files.go

```diff
@@ -185,6 +187,10 @@ func (fs *VFS) GenericOpen(name string) (File, error) {
 	return convertFile(func() (afero.File, error) { return fs.vfs.Open(name) }, func() error { return nil })
 }
 
+func OpenFile(name string, flag int, perm os.FileMode) (File, error) {
+	return globalFileSystem.OpenFile(name, flag, perm)
+}
```

Comment:
Is this function being used?

Reply:
not in this library but I think Graham was asking to have access as a filesystem global API

Reply:
I added a changelog entry to document that change and mentioned it in the PR description

Reply:
Thanks for adding this, we did want a simple function to call in Core-Tools.